core/src/main/kotlin/org/evomaster/core/problem/graphQl/FullType.kt

```diff
@@ -0,0 +1,17 @@
+class FullType {
+
+     var kind: __TypeKind?=null
+     var name: String?= null
+     var fields = ArrayList<__Field?>()
```

Comment:
we should avoid defining directly the types of list as their concrete instance.
This should rewritten as:
`var fields : MutableList<__Field> = mutableListOf()`
also notice that usually we should not allow `null` values inside a list (so the type of the list should be `__Field` and not `__Field?`)

Reply:
also notice that this comment applies to all other lists in these classes

Reply:
done